package.json

```diff
@@ -17,6 +17,7 @@
     "@svgr/webpack": "4.3.3",
     "@typescript-eslint/eslint-plugin": "^2.10.0",
     "@typescript-eslint/parser": "^2.10.0",
+    "axios": "^0.21.1",
```

Comment:
I want to avoid adding axios as a dependency. All other places use where http calls are needed `request`. Try sticking to that unless promises are absolutely required.

Reply:
I'll replace it with `window.fetch` then.

Reply:
I ended up using `ky` since it weighs very little and provides a small abstraction over window.fetch that will allow the code to maintain readability.

Reply:
I thought about it. I think you can stick with axios.